core/server/services/members/api.js

```diff
@@ -61,113 +62,90 @@ function validateMember({email, password}) {
     });
 }
 
-function parseMembersSettings() {
-    let membersSettings = settingsCache.get('members_subscription_settings');
-    if (!membersSettings) {
-        membersSettings = {
-            isPaid: false,
-            paymentProcessors: [{
-                adapter: 'stripe',
-                config: {
-                    secret_token: '',
-                    public_token: '',
-                    product: {
-                        name: 'Ghost Subscription'
-                    },
-                    plans: [
-                        {
-                            name: 'Monthly',
-                            currency: 'usd',
-                            interval: 'month',
-                            amount: ''
-                        },
-                        {
-                            name: 'Yearly',
-                            currency: 'usd',
-                            interval: 'year',
-                            amount: ''
-                        }
-                    ]
-                }
-            }]
-        };
-    }
-    if (!membersSettings.isPaid) {
-        membersSettings.paymentProcessors = [];
-    }
-    return membersSettings;
+function getSubscriptionSettings() {
+    return settingsCache.get('members_subscription_settings');
```

Comment:
🤔I think its ok to remove default from `parseSubscriptionSettings` in the new method, but we still need the `isPaid` conditional to set `paymentProcessors` as `[]`. It was added since by default `paymentProcessors` still exist in settings cache irrespective of the `isPaid` value, but we modify it as per our adapter settings. Lmk if I missed any other handling of it, otherwise just seems we need to add it back :)

Reply:
I've updated the default-settings to remove the default paymentProcessors, might still need some testing, but I don't see why we have stripe adapter as a default?

Reply:
I think removing it as default from `default-settings` is not enough if we also consider case where people already have member settings now in their db, unless we consider the case as fresh reset for all to use this now. 

Also think it was added in default for Admin to have the whole structure and just update the values, for ex when we switch between free and paid, we don't set all the default stripe adapter stuff there but just update either the `isPaid` value or the relevant stripe values when we change them in paid. If we remove the default and parsing here, we definitely need to update Admin to handle the case. I am ok if we do, but probably better to do it in a separate PR in that case. 😄